pulp_2to3_migration/app/plugin/rpm/pulp_2to3_models.py

```diff
@@ -415,3 +415,53 @@ def _get_profiles(profiles):
         """
         return ModulemdDefaults(module=self.module, stream=self.stream,
                                 profiles=self.profiles, digest=self.digest)
+
+
+class Pulp2Distribution(Pulp2to3Content):
+    """
+    Pulp 2to3 detail content model to store Pulp2 Distribution content details.
+    """
+
+    # Unit key fields
+    distribution_id = models.TextField()
+    family = models.TextField()
+    variant = models.TextField()
+    version = models.TextField()
+    arch = models.TextField()
```

Comment:
I wonder if that's enough.
How will you identify the location of all files which belong to the distribution tree.
I *think* you can get this info from `files` field in pulp2 but I'm not sure.